### DISCUSSION TITLE
Crash loop

### DISCUSSION THREAD

**Discussion by WishStone-Dragon** [Monika-After-Story/MonikaModDev#9381] [QA - tech support + general]
Hey folks,

how can I break a crash loop? 
TL;DR - I tried to select a Dev check in the Dev MAS and now every time I launch MAS, I crash.
    Edit: AH, I can IGNORE the error to get past it! 
Not fixed, but fixable by me.

Details: 
I THINK I finally got my sandbox fixed after patching. It's running, for the most part, but I started to run into a problem as I was trying out the Dev "Check Affection" commands.

`[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1541, in script call
    call call_next_event from _call_call_next_event_1
  File "game/event-handler.rpy", line 3119, in script call
    call expression _ev_list_item.evl from _call_expression
  File "game/dev/dev_affection-test.rpy", line 19, in script
    m 3eua "My affection as a number it would be...[persistent._mas_affection['affection']]."
  File "game/definitions.rpy", line 169, in get_field
    obj = obj[i]
KeyError: u"'affection'"

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1541, in script call
    call call_next_event from _call_call_next_event_1
  File "game/event-handler.rpy", line 3119, in script call
    call expression _ev_list_item.evl from _call_expression
  File "game/dev/dev_affection-test.rpy", line 19, in script
    m 3eua "My affection as a number it would be...[persistent._mas_affection['affection']]."
  File "C:\Users\Ges\PycharmProjects\MonikaModDev\Monika After Story\renpy\ast.py", line 613, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "C:\Users\Ges\PycharmProjects\MonikaModDev\Monika After Story\renpy\exports.py", line 1147, in say
    who(what, interact=interact)
  File "C:\Users\Ges\PycharmProjects\MonikaModDev\Monika After Story\renpy\character.py", line 868, in __call__
    what = what_pattern.replace("[what]", sub(what, translate=translate))
  File "C:\Users\Ges\PycharmProjects\MonikaModDev\Monika After Story\renpy\character.py", line 850, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Users\Ges\PycharmProjects\MonikaModDev\Monika After Story\renpy\substitutions.py", line 232, in substitute
    s = formatter.vformat(s, (), kwargs)
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 563, in vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 585, in _vformat
  File "game/definitions.rpy", line 169, in get_field
    obj = obj[i]
KeyError: u"'affection'"

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.9
[/code]
`

When restarting Moni, she is sad she crashed, then says "Where was I?" goes back into the last selected dialouge (Dev check affections) and falls right into the next crash.

Can I interrupt this?
